tools/protoxform/protoxform.py

```diff
@@ -38,16 +39,44 @@
 from envoy.annotations import resource_pb2
 from udpa.annotations import migrate_pb2
 
-CLANG_FORMAT_STYLE = ('{ColumnLimit: 100, SpacesInContainerLiterals: false, '
-                      'AllowShortFunctionsOnASingleLine: false, ReflowComments: false}')
-
 NEXT_FREE_FIELD_MIN = 5
 
 
 class ProtoXformError(Exception):
   """Base error class for the protoxform module."""
 
 
+def ExtractClangProtoStyle(clang_format_text):
```

Comment:
Would it be simpler to just bazel export .clang-format and just have it in the sandbox where it would be picked up automatically by the tool through recursive directory search? Up to you.

Reply:
This doesn't work due to the relative execute locations. The exported `.clang-format` appears as a sub-directory, not in the recursive parent hierarchy, inside the sandbox.

Reply:
This doesn't work due to the relative execute locations. The exported `.clang-format` appears as a sub-directory, not in the recursive parent hierarchy, inside the sandbox.